recipe/meta.yaml

```diff
@@ -11,29 +11,40 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python
+    - wheel
+    - setuptools
   run:
-    - python >=3.8
+    - python
     - boto3 >=1.16.35
     - smart_open
     - packaging
```

Comment:
```suggestion
```
I don't think `packaging` is used.

Reply:
only in 1 test, https://github.com/search?q=repo%3Aliormizr%2Fs3path%20packaging&type=code
but we skip tests, therefore, it might be not useful.

Reply:
I consider it unused even if used in a test. This is the runtime dependencies, not test dependencies.

Reply:
right, agree